frontend/src/stimulus/controllers/dynamic/work-packages/activities-tab/index.controller.ts

```diff
@@ -632,12 +634,13 @@ export default class IndexController extends Controller {
     }
   }
 
-  private quotedText(rawComment:string, userName:string) {
+  private quotedText(rawComment:string, userId:string, userName:string, textWrote:string) {
     const quoted = rawComment.split('\n')
       .map((line:string) => `\n> ${line}`)
       .join('');
 
-    return `${userName}\n${quoted}`;
+    // if we ever change CKEditor or how @mentions work this will break
+    return `<mention class="mention" data-id="${userId}" data-type="user" data-text="@${userName}">@${userName}</mention> ${textWrote}:\n\n${quoted}`;
```

Comment:
Ah, interesting. I'd imagine CKEditor as an internal builder class we can call? _*asking co-pilot, see if it can figure it out_ :)

Reply:
I thought it was an internal thing, did not think it was CKEditor things.

https://ckeditor.com/docs/ckeditor5/latest/api/module_mention_mentioncommand-MentionCommand.html

I'll try again a bit more.

Reply:
Somehow I cannot get this one to work, so I might go with the current implementation, and if needed we refactor as a next step. Don't think it's worth investing much more time into this one.